.travis.yml

```diff
@@ -77,6 +77,7 @@ script:
           -Dfinal.war.name=cbioportal \
           -Ddb.user=cbio_user \
           -Ddb.password=somepassword \
+          -Dehcache.xml.configuration=/ehcache.xml \
```

Comment:
why does this parameter need to be added - can't the default be e.g. `classpath:/ehcache.xml`?

Reply:
this line is no longer needed for travis testing because the cache is now disabled ... in addition, the default was available too